### PR TITLE
Remove old branding and dead links

### LINKED ISSUE #1190: xkit-extension.tumblr.com dead, several links not working
Broken links are:
- `bridge.js`
  - [L76](https://github.com/new-xkit/XKit/blob/1375a0e188e5bba3e85e4757506bbd1828f2ded4/Chrome/bridge.js#L76)
  - [L102](https://github.com/new-xkit/XKit/blob/1375a0e188e5bba3e85e4757506bbd1828f2ded4/Chrome/bridge.js#L102)
- `xkit_installer.js`
  - [L43](https://github.com/new-xkit/XKit/blob/1375a0e188e5bba3e85e4757506bbd1828f2ded4/Extensions/xkit_installer.js#L43)
- `xkit_patches.js`
  - [L26](https://github.com/new-xkit/XKit/blob/f16525cc99cb9b82d4400836bf1a1ca70a492577/Extensions/xkit_patches.js#L26)
- `people_notifier.js` (not actual link, just rebranding)
  - [L486](https://github.com/new-xkit/XKit/blob/71585beeed796a9411583f2bf78bff3851711e7c/Extensions/people_notifier.js#L486)
  - [L498](https://github.com/new-xkit/XKit/blob/71585beeed796a9411583f2bf78bff3851711e7c/Extensions/people_notifier.js#L498)
- `old_notifications.js` (same as above)
  - [L4](https://github.com/new-xkit/XKit/blob/1375a0e188e5bba3e85e4757506bbd1828f2ded4/Extensions/old_notifications.js#L4)
- `servant.js` (same as above)
  - [L122](https://github.com/new-xkit/XKit/blob/1375a0e188e5bba3e85e4757506bbd1828f2ded4/Extensions/servant.js#L122)
  - [L150](https://github.com/new-xkit/XKit/blob/1375a0e188e5bba3e85e4757506bbd1828f2ded4/Extensions/servant.js#L150)

### PR DESCRIPTION
**Pull request by Wolvan**
http://new-xkit-extension.tumblr.com/ is no longer available, a few
parts of XKit were still linking to it. This removes or replaces most of
those instances with new-xkit-extension

Closes #1190

### CODE REVIEW

**Comment by ThePsionic** [comment]
Our blog definitely still exists :P I think you went on autopilot there.
While you're at it, `icon_nots.js` still uses ancient versioning, and likely old branding too.

**Comment by Wolvan** [comment]
Our blog exists, but it doesn't have whatever posts the old xkit blog had. The posts described the problem and a solution, since we don't have that, I thought it might be a better idea to just say, "hey, come to support chat and we can try and help you resolve this. Makes sense?"

**Comment by ThePsionic** [comment]
>http://new-xkit-extension.tumblr.com/ is no longer available

**Comment by Wolvan** [comment]
You know what I mean, Psi

**Comment by ThePsionic** [comment]
Also, useful commentary from @nightpool at #1190 is useful:
>xkit-extension-blog.tumblr.com has all of the old pages

**Comment by ThePsionic** [comment]
I assume Chrome needs a new build once the bridge is updated?

**Comment by Wolvan** [comment]
It would, yes

File: Extensions/xkit_installer.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ XKit.extensions.xkit_installer = new Object({
...
 			XKit.installed.remove("xkit_installer");
-			XKit.window.show("Thanks for installing me!","<b>Installation of XKit is complete.</b><br/>After refreshing the page, the X icon will show up on the header next to the Inbox button. From there, you can customize XKit to your liking and get new extensions.", "info","<a href=\"http://www.tumblr.com/dashboard\" id=\"xkit-restart-and-use\" class=\"xkit-button default\">Refresh page and start using XKit</a><a href=\"http://xkit-extension.tumblr.com/post/52742121604/chrome-system-restores-corrupt-profile-settings-and\" class=\"xkit-button\">XKit keeps re-installing itself?</a>");
+			XKit.window.show("Thanks for installing me!","<b>Installation of XKit is complete.</b><br/>After refreshing the page, the X icon will show up on the header next to the Inbox button. From there, you can customize XKit to your liking and get new extensions.", "info","<a href=\"http://www.tumblr.com/dashboard\" id=\"xkit-restart-and-use\" class=\"xkit-button default\">Refresh page and start using XKit</a><a href=\"https://new-xkit-support.tumblr.com/\" class=\"xkit-button\">XKit keeps re-installing itself? Join our Livesupport</a>");
```
Comment (nightpool) [inline Extensions/xkit_installer.js]: Livesupport isn't a word

**Pull request closed without merging**